tests/test_end_to_end.py

```diff
@@ -149,11 +159,16 @@ def test_simple_database_to_docx_files(tmpdir):
             }
         ]
     }
-    status = run_n2y(tmpdir, config)
+    status = run_n2y(tmp_path, config)
     assert status == 0
-    output_directory = os.path.join(tmpdir, "database")
-    generated_files = {f for f in listdir(output_directory) if isfile(join(output_directory, f))}
-    assert generated_files == {"A.docx", "B.docx", "C.docx"}
+    for expected_filename in ["A.docx", "B.docx", "C.docx"]:
+        expected_path = tmp_path / "database" / expected_filename
+        doc = pandoc.read(expected_path.read_bytes(), format="docx")
+        metadata_dict = doc[0][0]
+
+        # Should arrive from yaml
```

Comment:
This is where I expected a comment -- does each record in a simple database inherit these values? I assumed so, and maybe it's harmless.